src/components/Avatar/Avatar.module.css

```diff
@@ -0,0 +1,48 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # AVATAR
+\*------------------------------------*/
+
+/**
+ * Avatar
+ */
+.avatar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  overflow: hidden;
+
+  color: var(--eds-theme-color-text-neutral-strong);
+  background-color: var(--eds-theme-color-background-neutral-medium);
+}
+
+.avatar:focus {
```

Comment:
Should this be [focus-visible](https://github.com/chanzuckerberg/edu-design-system/blob/main/docs/CODE_GUIDELINES.md?plain=1#L108-L126)?